### ISSUE TITLE
Segfault when loading scene using invalid node type with children nodes (e.g. AspectRatioContainer in 3.2.3)

### ISSUE THREAD

**Issue by ShatReal** [godotengine/godot#46404]
**Godot version:**

3.2.3

**OS/device including version:**

Windows 10

**Issue description:**

Opening project with AspectRatioContainer in 3.2.3 crashes the engine.

**Steps to reproduce:**
1. Make project with AspectRatioContainer in 3.2.4
2. Open project in 3.2.3

**Minimal reproduction project:**

**Comment by akien-mga** [comment]
That's normal, this node doesn't exist in 3.2.3.

**Comment by ShatReal** [comment]
I see, someone on the Godot forums said I should report this

**Comment by akien-mga** [comment]
Can you clarify what you mean by crash? I can't reproduce any crash here, it's handled as I expect it be, with an error:
```
ERROR: is_class_enabled: Cannot get class 'AspectRatioContainer'.
   At: core/class_db.cpp:1329.
```
And the scene which contained this node loads fine, with the AspectRatioContainer node removed.

**Comment by ShatReal** [comment]
What I mean is that Godot just closes when I try to open the project.

**Comment by akien-mga** [comment]
I can't reproduce that, could you provide a minimal reproduction project?

In any case, I assume that it's not crashing but simply closing with errors because the project files are invalid and cannot be opened by Godot 3.2.3. Those errors should make it explicit that a scene with an unknown node name was attempted to load and this couldn't succeed (and without anything to load, the engine will close).

Something like this:
```
SCRIPT ERROR: GDScript::reload: Parse Error: Unknown class: "AspectRatioContainer"
   At: res://AspectRatioContainer.gd:1.
ERROR: reload: Method failed. Returning: ERR_PARSE_ERROR
   At: modules/gdscript/gdscript.cpp:599.
ERROR: is_class_enabled: Cannot get class 'AspectRatioContainer'.
   At: core/class_db.cpp:1329.
```
(Though again for me that doesn't actually stop the application, it still runs fine, without that node.)

**Comment by ShatReal** [comment]
My aspectratiocontainer has a child.
[test.zip](https://github.com/godotengine/godot/files/6044687/test.zip)

**Comment by akien-mga** [comment]
Thanks, with this project I do get a segmentation fault crash:
```
ERROR: is_class_enabled: Cannot get class 'AspectRatioContainer'.
   At: core/class_db.cpp:1329.

Thread 1 "godot" received signal SIGSEGV, Segmentation fault.
0x000000000195c01d in Node::get_node_or_null (this=0x0, p_path=...) at scene/main/node.cpp:1320
1320            ERR_FAIL_COND_V_MSG(!data.inside_tree && p_path.is_absolute(), NULL, "Can't use get_node() with absolute paths from outside the active scene tree.");

(gdb) bt
#0  0x000000000195c01d in Node::get_node_or_null (this=0x0, p_path=...) at scene/main/node.cpp:1320
#1  0x0000000001ff4d89 in SceneState::instance (this=0x52ca3b0, p_edit_state=SceneState::GEN_EDIT_STATE_DISABLED) at scene/resources/packed_scene.cpp:96
#2  0x0000000001ff711d in PackedScene::instance (this=0x4f22260, p_edit_state=PackedScene::GEN_EDIT_STATE_DISABLED) at scene/resources/packed_scene.cpp:1698
#3  0x0000000000971b93 in Main::start () at main/main.cpp:1942
#4  0x00000000008f9979 in main (argc=<optimized out>, argv=<optimized out>) at platform/x11/godot_x11.cpp:55
```

That can't be fixed in 3.2.3-stable retroactively but we should likely prevent such crash for future releases.
